app/main.py

```diff
@@ -1 +1,30 @@
-# write your code here
+from __future__ import annotations
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf_ranger import ElfRanger
+from app.players.elves.druid import Druid
+from app.players.dwarves.dwarf_warrior import DwarfWarrior
+from app.players.dwarves.dwarf_blacksmith import DwarfBlacksmith
+
+
+def calculate_team_total_rating(
+        team: list
+        | ElfRanger
+        | Druid
+        | DwarfWarrior
+        | DwarfBlacksmith
```

Comment:
there is an incorrect type annotation - u are annotate that the team variable can be a list, Elfranger or druid or ... but it will be always list - u should write a type annotation for what can be inside a list f.e list[Player] - there will always be a list with instances of Player class